models/DataObject/AbstractObject.php

```diff
@@ -999,9 +1000,7 @@ public function setId($o_id)
     public function setParentId($o_parentId)
     {
         $o_parentId = (int) $o_parentId;
-        if ($o_parentId != $this->o_parentId && $this instanceof DirtyIndicatorInterface) {
-            $this->markFieldDirty('o_parentId');
-        }
+        $this->checkFieldDirty('o_parentId', $o_parentId);
```

Comment:
still needed ?

Reply:
just future-proof :)

Reply:
Hmm, I'm wondering ... it's just used here, can't we avoid it? Then we can kick the additional method.

Reply:
kicked it ;)

Reply:
Nice, thanks!